security/pkg/nodeagent/sds/plugin/mockintetestcredfetcher.go

```diff
@@ -0,0 +1,40 @@
+package plugin
```

Comment:
Please combine this mock credfetcher into security/pkg/credentialfetcher/plugin/mock.go, we need to maintain only one mock credfetcher for testing. sds/plugin/ holds the token exchange plugin which is different.